docs/menu.rb

```diff
@@ -0,0 +1,35 @@
+class Menu  
+
+    attr_reader :order
+
+    def initialize
+       @list = { 'Latte' => 2.90, 'Cappuccino' => 2.80, 'Cake' => 3.50, 'Sandwich' => 4.00 }    
+
+       @order = []
+    end  
+
+    def list_menu   
+        @list.each do |key, value|
+            puts "Item #{key} Cost #{value}"
+        end
+    end  
+
+    def select_item(item, num) 
+        if num >= 1 || num == nil
+            @order << { item => num }   
+            puts "#{num}x #{item} have been added to your basket."   
+        else 
+            puts "Plaese input a number of items."
```

Comment:
What you could do here is to set num to equal 0 in the argument. This would reduce the code used as you wouldnt need to account for the number being nill and there wouldn't be a need for the if statement.